File: ANDROID APPLICATIONS/Xamarin/2014/KANDOU v2.5/KANDOU v1/DataTypes/KanjiDataType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace KANDOU_v1
{
    [Serializable]
    class KanjiDataType
    {
        public string sign;
        public string meaning;
        public string reading;

        public SubmissionOfKanji[] submissions;

        public KanjiDataType()  { }

        public KanjiDataType(string sign, string meaning, string reading, SubmissionOfKanji[] submissions)
        {
            this.sign = sign;
            this.meaning = meaning;
            this.reading = reading;
            this.submissions = submissions;
        }
    }
}

File: ANDROID APPLICATIONS/2015/Kandou mobile v3 (beta)/KANDOUmobile_V3/src/aplication/data/KanjiDataType.java
package aplication.data;

import java.io.Serializable;

@SuppressWarnings("serial")
public class KanjiDataType implements Serializable{
	public String sign;
    public String meaning;
    public String reading;

    public SubmissionOfKanji[] submissions;

    public KanjiDataType()  { }
    
    public KanjiDataType(String sign, String meaning, String reading, SubmissionOfKanji[] submissions)
    {
        this.sign = sign;
        this.meaning = meaning;
        this.reading = reading;
        this.submissions = submissions;
    }
}
